doc/source/whatsnew/v0.19.0.txt

```diff
@@ -613,3 +613,8 @@ Bug Fixes
 - Bug in ``groupby`` with ``as_index=False`` returns all NaN's when grouping on multiple columns including a categorical one (:issue:`13204`)
 
 - Bug where ``pd.read_gbq()`` could throw ``ImportError: No module named discovery`` as a result of a naming conflict with another python package called apiclient  (:issue:`13454`)
+
+- Compatibility with maptlolib 2.0 (:issue:`13333`)
```

Comment:
As changes are only for tests, there is no bugs related to mpl2. Not sure on what section the description should be...